### PR TITLE
add linkerd to service discovery section

### PR DESCRIPTION
**Pull request by siggy**
## What type of changes does your doc introduce?
- [ ] Bug fix
- [x] New feature 

## What is the urgency level of this change?
- [ ] Blocker
- [ ] High
- [x] Medium

## I have completed these items:
- [x] I have tested all commands and code snippets.
- [x] I have built locally and tested for broken links and formatting.
- [ ] I have made this change for all affected versions (e.g. 1.7, 1.8, and 1.9).
- [x] My doc follows the style guide: https://github.com/dcos/dcos-docs#contributing.

Ping `@emanic`, `@sascala`, or `@joel-hamill` for reviewing pull request changes.

### CODE REVIEW

File: 1.8/usage/service-discovery/third-party-solution/linkerd/index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+![diagram](img/diagram.png)
+
+In addition to service discovery and resilient communication, linkerd provides a uniform layer of visibility across all services and a convenient service metrics dashboards.
+
+## Next Steps
+
+- Install linkerd and linkerd-viz by following the [linkerd Tutorial][2].
+- For a step-by-step example with a sample app, read the blog post: [DC/OS Blog: Service Discovery and Visibility with Ease on DC/OS][3]
+
+ [1]: https://linkerd.io
```
Comment (ryadav88) [inline 1.8/usage/service-discovery/third-party-solution/linkerd/index.md]: remove the link here. should point to a location in DC/OS docs.

**Review by ryadav88 — CHANGES_REQUESTED** [review]

File: 1.8/usage/service-discovery/third-party-solution/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+menu_order: 300
+---
+
+In addition to [VIPs][1], [Minuteman][2], [Marathon-LB][3], and [Mesos-DNS][4], the DC/OS Universe provides other third party solutions intended for more specific use cases.
+
+# linkerd
+
+[linkerd][5] is a service mesh that does load balancing, routing, circuit breaking, and retries. linkerd is a good option when you have latency-sensitive HTTP, gRPC or Thrift communication between services, and can direct communication through proxy.
+
+You can use linkerd to route HTTP, Thrift, and gRPC traffic through a service mesh by their logical name by following the instructions in the [linkerd Tutorial][6] section. By default, linkerd uses Marathon task names as logical names, but can also use dedicated service discovery systems such as ZooKeeper, Consul or etcd.
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/third-party-solution/index.md]: Lowercase `Tutorial`

File: 1.8/usage/service-discovery/third-party-solution/linkerd/index.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
+---
+post_title: linkerd
+menu_order: 0
+---
+
+linkerd is a service mesh for DC/OS. It is installed on every agent, and acts as a transparent, load-balancing proxy, providing service discovery and resilient communication between services.
+
+The linkerd Universe package is configured to use Marathon for service discovery. This means that applications and services can refer to each other by their Marathon task name. For example, a connection to `http://myservice` made through linkerd will be load-balanced over instances of the Marathon application `myservice`, without using DNS. linkerd can also be configured to use dedicated service discovery systems such as ZooKeeper, Consul or etcd (as well as DNS itself) and to failover between these systems and/or Marathon.
```
Comment (joel-hamill) [inline 1.8/usage/service-discovery/third-party-solution/linkerd/index.md]: - Change `linkerd can also be configured` to `You can also configure linkerd`
- Change `systems and/or Marathon` to `systems and Marathon` (https://americaneditor.wordpress.com/2013/03/11/the-drama-of-and-and-or/)

Comment (siggy) [inline 1.8/usage/service-discovery/third-party-solution/linkerd/index.md]: TIL The Drama of “And” and “Or” :D

**Review by ryadav88 — APPROVED** [review]

**Pull request merged**